.nvmrc

```diff
@@ -0,0 +1 @@
+v14.8.0
```

Comment:
Currently I'm the only one using this! Codespaces changes my node version _constantly_, this works nicely with `nvm use` to set properly.